openedx/features/partners/views.py

```diff
@@ -9,3 +14,18 @@ def dashboard(request, slug):
     if partner.slug == 'give2asia':
         return g2a_dashboard(request)
 
+
+@require_http_methods(["POST"])
+@sensitive_post_parameters('password')
+@csrf_exempt
+def register_user(request, slug):
+    """
+    This is general registering view, for users of all partners
+    :param request: The Django request.
+    :param slug: partner slug
+    :return: JsonResponse object with success/error message
+    """
+    partner = get_object_or_404(Partner, slug=slug)
+    # TODO: we need to make it generic for all partners
```

Comment:
@emraan004 @ahmed-faraz46 

Whenever we do this, firstly we have to change it here as well. Moreover we might be able to use importlib.import_module here

Reply:
This is being handled by @ahmed-faraz46 in another PR